source/changelog.txt

```diff
@@ -12,6 +12,24 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.0
+------
+
+*Released August 16, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1550` - Adds support for Queryable Encryption Range queries  
```

Comment:
We probably at least also want to mention:
- Support for proxies in environment variables
- The removal of --tlsUseSystemCA
- That this removes support for the Range Preview version
- That this removes support for out-of-the-box support for automatic encryption on deprecated operating systems
- The new command line flags added for OIDC support

Reply:
I've updated the features list -- thanks! What issue does "removes support for out-of-the-box support for automatic encryption on deprecated Linux operating systems" correspond to?

Thank you!

Reply:
> What issue does "removes support for out-of-the-box support for automatic encryption on deprecated Linux operating systems" correspond to?

I'm no expert but if I'd have to guess it is probably a side-effect of bumping the node-driver or something like libmongocrypt.